/**:
  # Global Parameters
  ros__parameters:
    tf_timeout_threshold: 0.05

/**:
  # Basic Robotics Functions
  platform:
    ros__parameters:
      cmd_freq: 30.0  # Hz of platform commands send

  # State Estimator
  state_estimator:
    ros__parameters:
      plugin_name: "raw_odometry"
      base_frame: "base_link"
      global_frame: "earth"
      odom_frame: "odom"
      enable_tf: true

  # Motion Controller
  controller_manager:
    ros__parameters:
      plugin_name: "pid_speed_controller"
      cmd_freq: 30.0 # Hz of controller commands send
      info_freq: 10.0

  # Behaviors Motion
  TakeoffBehavior:
    ros__parameters:
      takeoff_plugin_name: "takeoff_plugin_speed"
  LandBehavior:
    ros__parameters:
      land_plugin_name: "land_plugin_speed"
  GoToBehavior:
    ros__parameters:
      go_to_plugin_name: "go_to_plugin_position"
  FollowPathBehavior:
    ros__parameters:
      follow_path_plugin_name: "follow_path_plugin_position"
